src/opnsense/service/templates/OPNsense/Proxy/parentproxy.conf

```diff
@@ -0,0 +1,27 @@
+{% if helpers.exists('OPNsense.proxy.general.parentproxy.enabled') and OPNsense.proxy.general.parentproxy.enabled == '1' %}
```

Comment:
Normally what require Y does is it enforces this particular setting so we don’t have to tiptoe around helpers.exist all the time in the template. The same goes for constraints tied to this setting.

Reply:
True, sorry